### PR TITLE
added navigator with transition button

### PR DESCRIPTION
**Pull request by ochanje210**

### CODE REVIEW

**Comment by mun625** [comment]
Overall looks good.
 1. But I think 'first.js' and 'seond.js' is not good name. I mean, I can't understand what's their role     when hear its name.
 2. And they should be component. not container.
other things are great.

**Comment by ochanje210** [comment]
I agree that 'first' and 'second' are not good names. I just followed up the tutorials and ended up having those names. Do you have any ideas?
Also, why do u think they are component again lolol

**Comment by yuda1124** [comment]
LGTM

File: shared/containers/first.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      <View style={{flex: 1}}>
+        <View style={{flex: 1}}>
+          <HeaderLayout
+            onForward={ () => {
+              const nextIndex = this.props.route.index + 1;
+              this.props.navigator.push({
+                index: nextIndex
+              });
+            }
+            }
```
Comment (shberry16) [inline shared/containers/first.js]: Indentation looks a bit weird here. Can you take a look? :)

File: shared/containers/headerLayout.js [hunk 2]
```diff
@@ -3,22 +3,22 @@ import {update_markers} from '../actions/mapActions';
 import Headerbox from '../components/headerbox';
 
 const mapStateToProps = () => {
-    return {
-        //TBD
-    };
+  return {
+    // TBD
```
Comment (shberry16) [inline shared/containers/headerLayout.js]: Prefer using // TODO(ochanje210): Implement something.

**Comment by mun625** [comment]
Cause 'first' and 'second's role is only presenting view.

File: shared/components/map.js [hunk 3]
```diff
@@ -46,7 +46,7 @@ export default class Map extends Component {
       <View style ={styles.container}>
         <MapView
           style ={styles.map}
-          initialRegion ={this.props.currentLocation}
+          region ={this.props.currentLocation}
```
Comment (mun625) [inline shared/components/map.js]: This was the point

Comment (ochanje210) [inline shared/components/map.js]: great work

**Comment by ochanje210** [comment]
LGTM

**Pull request merged**